[OT276-37] Endpoint datos usuario autenticado.

EmaGonzalezxd opened the pull request:
## Why is this change required?

### Motivation and Summary
<!--
- JIRA Ticket, User Story, etc
-->

### Type of change
- [ x] New feature
- [ ] Bug fix
- [ ] New Tests (no functional change)
- [ ] Refactor (no functional change)

### Checklist

- [ x] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

EmaGonzalezxd committed:
OT276-37

Creación de las clases UserController, UserControllerMapper y UserResponse.

EmaGonzalezxd committed:
Update UserController.java

Metodo Get agregado.

EmaGonzalezxd committed:
feature/OT276-37

Endpoint datos usuario autenticado.

EmaGonzalezxd committed:
Merge branch 'main' into feature/OT276-37

EmaGonzalezxd committed:
OT276-37

EmaGonzalezxd committed:
Update AuthController.java

EmaGonzalezxd committed:
Update application.properties

manerajona commented on src/main/java/com/alkemy/ong/ports/input/rs/controller/UserController.java:
```diff
@@ -22,6 +24,12 @@ public class UserController {
     private final UserService service;
     private final UserControllerMapper mapper;
 
+    @GetMapping("/me")
```
Va solo en `Auth` revertir estos cambios por favor.

manerajona replied:
@EmaGonzalezxd este metodo va en `AuthController` no en `UserController`

manerajona commented on src/main/java/com/alkemy/ong/ports/input/rs/api/UserApi.java:
```diff
@@ -15,4 +18,6 @@ public interface UserApi {
 
     ResponseEntity<?> register(@Valid @RequestBody CreateUserRequest userRequest);
+
+    ResponseEntity<UserResponse> getUserInformation(@AuthenticationPrincipal User user);
```
Este método es parte de `AuthApi`

EmaGonzalezxd committed:
Update UserApi.java

EmaGonzalezxd committed:
Update UserController.java

Ese metodo ya estaba en el AuthController pero tambien estaba en el UserController.

manerajona committed:
UserApi def
